powersimdata/scenario/execute.py

```diff
@@ -189,36 +191,43 @@ def _launch_on_server(self, threads=None, extract_data=True):
             # Use the -t flag as defined in call.py in REISE.jl
             extra_args.append("--threads " + str(threads))
 
+        if solver:
+            extra_args.append("--solver", solver)
```

Comment:
Since we test the type of `extract_data` should we do the same with `solver` and `threads`?

Reply:
We check `threads` just before calling this, since it's also passed to `_launch_in_container`. I can add the check for `solver` though